ProcessList.c

```diff
@@ -172,7 +179,7 @@ static void ProcessList_updateTreeSetLayer(ProcessList* this, unsigned int left_
    // Thus, if present, set the right_bound to the last process on the layer
```

Comment:
```suggestion
   // Thus, if present, set the `rightBound` to the last process on the layer
```
Probably needs to be changed in some more places inside comments.